### DISCUSSION TITLE
Setting hostname for easy navigation

### DISCUSSION THREAD

**Discussion by Addonnis** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#207] [Ideas]
The default hostname for the ESP is ESP-XXXXXX (I think the digits are the last characters of the ESP mac address)

You can set a custom hostname by adding a line on 267 of main.cpp in the format:
`WiFi.hostname("Name");`

This line must be before Wifi.begin.

You can then hit the web interface by going to name.local, which is family friendly.

![image](https://user-images.githubusercontent.com/97511696/151682210-f77028d5-4991-4e20-97f7-b929d906f033.png)

![image](https://user-images.githubusercontent.com/97511696/151682257-373f427d-529c-4dd1-8b37-f238a0a1b60f.png)

**Comment by torei** [comment]
we are already using that on line 255.
there is even a config for that (config.h:15).
:-)

**Comment by Handy-Harry** [comment]
http://layzspa.local/ works for me out of the box. But on my Unifi router I get the old hostname espxxxx as the hostname.
When I put WiFi.hostname(netHostname); after  WiFi.begin(apSsid, apPwd); the correct hostame is displayed in my routers UI.

**Reply by visualapproach** [reply]
Does not work on my router :-(
But works in browser :-)
